src/Confluent.Kafka/PartitionRequest.cs

```diff
@@ -0,0 +1,98 @@
+﻿//    Copyright 2019 Chad Jessup
```

Comment:
The copyright will need to be Confluent's.

Reply:
Thought as much, the LICENSE file in the repo was a little ambiguous, and I didn't want to claim copyright for a 3rd party without some representative stating as much.